projects/extension/src/background/ConnectionManagerWithHealth.ts

```diff
@@ -59,7 +59,7 @@ export interface ChainInfo<SandboxId> {
    * This value should be treated as a hint, and not a strong information. It is possible for
    * the value to be erroneous, for example for chains that have tweaked their header format.
    */
-  latestBestBlockHeight?: number
+  bestBlockHeight?: number
```

Comment:
Why the rename?

Reply:
It felt only right to keep naming consistent across and I preferred the shorter-length name

Reply:
I don't mind

Reply:
I'll keep it as is then